docs/source/api_reference/tile_layer.rst

```diff
@@ -61,4 +61,5 @@ opacity            1.0
 visible            True
 no_wrap            False
 show_loading       False
+loading            False (dynamically updated)
```

Comment:
This makes me think that the `loading` attribute should be specified as read-only. Something like:

```python
loading = Bool(False, read_only=True).tag(sync=True)
```

So that users would have an error when they try to assign to it. Could be in a separate PR though, as you wish.

Reply:
Yes, I though of that. I'll do it right now and see if doesn't break anything. If it doesn't, then I'll commit again.

Reply:
`Cannot find module "leaflet.awesome-markers"` in JS console... :thinking:

Reply:
Oh yes, it's a new dependency that has been added. You need to run `npm install` to install it.